libsolidity/interface/CompilerStack.cpp

```diff
@@ -1024,6 +1025,39 @@ Json::Value CompilerStack::methodIdentifiers(string const& _contractName) const
 	return methodIdentifiers;
 }
 
+Json::Value CompilerStack::errorIdentifiers(string const& _contractName) const
+{
+	if (m_stackState < AnalysisPerformed)
+		solThrow(CompilerError, "Analysis was not successful.");
+
+	Json::Value errorIdentifiers(Json::objectValue);
+	for (auto const& it: contractDefinition(_contractName).interfaceErrors())
+	{
+		string signature = it->functionType(true)->externalSignature();
```

Comment:
`true` means that you are requesting an internal function type. Does this actually work with `true`?
```suggestion
		string signature = it->functionType(false)->externalSignature();
```

Reply:
It does work, but after taking a deeper look, I should probably change it for type?

https://github.com/ethereum/solidity/blob/a07b3ec70f91fdbf35929d5d474e91e739f792e8/libsolidity/ast/AST.cpp#L533-L544